qa/rolling-upgrade/build.gradle

```diff
@@ -74,6 +74,7 @@ for (Version bwcVersion : BuildParams.bwcVersions.wireCompatible) {
     }
     systemProperty 'tests.upgrade_from_version', bwcVersionStr
     systemProperty 'tests.rest.suite', 'old_cluster'
+    systemProperty 'tests.repo_location', "${buildDir}/cluster/shared/repo/${baseName}/test"
```

Comment:
Do we need to cleanup repository directory like we are doing for `path.repo` in doFirst block

Reply:
The cleanup cleans the entire folder structure. I am utilizing a subfolder within the actual repo which should be cleaned up by the existing code.